client/state/current-user/gravatar-status/test/actions.js

```diff
@@ -42,22 +42,22 @@ describe( 'actions', () => {
 
 	describe( '#uploadGravatar', () => {
 		it( 'dispatches request action when thunk triggered', () => {
-			uploadGravatar( 'file', 'bearerToken', 'email' )( spy );
+			uploadGravatar( 'file', 'email' )( spy );
 			expect( spy ).to.have.been.calledWith( sinon.match( {
 				type: GRAVATAR_UPLOAD_REQUEST
 			} ) );
 		} );
 
 		describe( 'successful request', () => {
 			useNock( ( nock ) => {
-				nock( 'https://api.gravatar.com' )
+				nock( 'https://public-api.wordpress.com' )
 					.persist()
-					.post( '/v1/upload-image' )
+					.post( '/wpcom/v2/gravatar-upload' )
 					.reply( 200, 'Successful request' );
 			} );
```

Comment:
note: with the use of `http()` in the data layer we should be able to completely eliminate the use of `nock()` - just make sure that the dispatched action has the right path and properties, etc…